InvenTree/company/urls.py

```diff
@@ -4,18 +4,12 @@
 
 from . import views
 
-company_detail_urls = [
-
-    re_path(r'^thumb-download/', views.CompanyImageDownloadFromURL.as_view(), name='company-image-download'),
```

Comment:
Suggestion: So this is prob. breaking, we should tag the PR in the release notes